### PR TITLE
Added the disclaimer to not use AWS CF templates in Production

### PR DESCRIPTION
**Pull request by kevinmiller-conley**
## Description
https://jira.mesosphere.com/browse/DOCS-2362
## Urgency
- [ ] Blocker 
- [ ] High
- [X] Medium

## Requirements
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: 1.11/installing/ent/cloud/aws/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ excerpt:
 enterprise: true
 ---
 
-You can create a DC/OS cluster for Amazon Web Services (AWS) by using the DC/OS templates on AWS CloudFormation.
+You can create a DC/OS cluster for Amazon Web Services (AWS) by using the DC/OS templates on AWS CloudFormation. (Due to missing upgrade capabilities, they should not be used in Production)
```
Comment (stbof) [inline 1.11/installing/ent/cloud/aws/index.md]: Due to the following limitations you shouldn't use these templates in production:
- Deployment errors are difficult to troubleshoot.
- The templates cannot be modified.
- DC/OS version of clusters created with the templates cannot be upgraded.

For production clusters, Mesosphere recommends you use the methods described in [DC/OS Open Source Advanced  Installation Guide](/1.11/installing/oss/custom/advanced/) or [DC/OS Enterprise Advanced Installation Guide](/1.11/installing/ent/custom/advanced/).

Please propagate to all files.

**Comment by karlkfi** [comment]
The note format isn't visible enough. I think we can use the new Warning Message shortcode: https://github.com/mesosphere/dcos-docs-site/blob/master/pages/test/index.md#message-data-hidetrue

**Comment by kevinmiller-conley** [comment]
@karlkfi @jenafermiller @justinrlee @stbof I hope this latest commit addresses the concerns.

File: 1.11/installing/ent/cloud/aws/index.md [hunk 2]
```diff
@@ -21,4 +21,13 @@ Choose the advanced templates if you want to customize the AMI, VPC, or Instance
...
-* More setup work is required
\ No newline at end of file
+* More setup work is required
+
+[message type="warning"] DC/OS CloudFormation templates are intended for reference only and are not recommended for production use due to the following limitations:
+
+- CloudFormation does not allow for coordinated zero-downtime in-place updates within Auto Scaling groups.
+- CloudFormation does not allow for automated zero-downtime replacement of Auto Scaling groups.
+- Replacing DC/OS agent nodes requires manual data migration of local storage volumes for stateful services.
+- Updates of DC/OS on AWS CloudFormation has not been automated, validated, or documented.
```
Comment (karlkfi) [inline 1.11/installing/ent/cloud/aws/index.md]: Got a problem with verb noun agreement here. "updates" and "has" don't go together. 

Either change "has" to "have" or change "updates of" to "updating".

Change needs to be made on all copies.

**Comment by karlkfi** [comment]
Should this warning be at the top under the description instead of at the bottom?

**Comment by kevinmiller-conley** [comment]
I think given that the page is only about three paragraphs long, it's perfectly visible at the bottom.

**Comment by karlkfi** [comment]
@benh asked that we link to the recommended solution for production use. So maybe we need a sentence like...

> The recommended way to install production ready DC/OS that can be upgraded in-place is to use the Advanced Installer.

**Comment by kevinmiller-conley** [comment]
Updated to include the link.

**Pull request merged**